tests-legacy/Endpoints/AbstractEndpointAdminTest.php

```diff
@@ -88,13 +86,13 @@ protected function employeeLogin()
      */
     private function initContainerInstance()
     {
-        $kernel = new AppKernel('test', true);
-        $kernel->boot();
-
-        $reflectedClass = new ReflectionClass(SymfonyContainer::class);
-        $instanceProperty = $reflectedClass->getProperty('instance');
-        $instanceProperty->setAccessible(true);
-        $instanceProperty->setValue($kernel->getContainer());
-        $instanceProperty->setAccessible(false);
+        $newKernel = new AppKernel('test', true);
+        $newKernel->boot();
+
+        // Make kernel accessible for SymfonyContainer
```

Comment:
🤔

Reply:
Yes, `SymfonyContainer` is a bit weirdly done but it expects a global $kernel variable or it can't access to it So in test environment we need to update this global variable each time a kernel is booted

And I know it shouldn't be related with integration tests, but at some point this was failing so I had no choice but to fix it

Reply:
It might be because the kernel now clears its references, but it's cleaner to do it and it was necessary for many other use cases So just this one side effect is acceptable besides it's clear to do it this way instead of cheating with reflection to set a private field